src/Mod/PartDesign/Gui/TaskHoleParameters.ui

```diff
@@ -227,62 +185,33 @@
      </property>
     </widget>
    </item>
-   <item row="11" column="1">
-    <widget class="QComboBox" name="ThreadSize">
-     <property name="sizePolicy">
-      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
-       <horstretch>0</horstretch>
-       <verstretch>0</verstretch>
-      </sizepolicy>
+   <item row="26" column="1">
+    <widget class="Gui::PrefQuantitySpinBox" name="TaperedAngle">
```

Comment:
This spinbox causes a column of whitespace to appear for me (also in Master, not caused by this PR).
Simplest would be to constrain the spinbox max width to the same value as the ThreadClass combobox.
![image](https://user-images.githubusercontent.com/1278189/104836271-1895f780-58ad-11eb-9bb4-5531209af031.png)

Reply:
> This spinbox causes a column of whitespace to appear for me

I have no clues why you get a different output than I do. Here is what I get with this PR:

![FreeCAD_LXXqyVid2G](https://user-images.githubusercontent.com/1828501/104863859-956db380-5937-11eb-9fcc-a2ba0d5c84e7.png)

I checked the tapered angle field and it has the same settings than the dimension combobox and thus bot appear for me perfectly aligned, also when I change the width of the dialog in FC.
However, since you also see this in master this PR does not make it worse, in contrary, it fixes other UI issues for you.

Reply:
Of course this is not blocking this PR. I was only hoping it could be fixed too.

Reply:
The Size combobox has max width = 140
![image](https://user-images.githubusercontent.com/1278189/104913034-58341080-598d-11eb-8d90-4e1c43386a51.png)

The TaperANgle combobox has max with of 1600000
![image](https://user-images.githubusercontent.com/1278189/104913146-7c8fed00-598d-11eb-8f66-ee938d5c64ea.png)

This is the root cause for sure.

Reply:
> This is the root cause for sure.

This is strange because but have the policy Minimum for the width. So despite the tapered angle can run wider it does not need to.
However, I removed now the 140 px limit for the comboboxes. Please test if this fixes the issue for you.

Reply:
I can't speak to whether this fixes it for @davidosterberg, but mine looks like yours now, @donovaly. Perhaps a later PR could clean up what still looks to me to be some extra space in that column. At least in English, "Countersink Angle" is the longest thing in it, and has quite a bit of extra space after it.

Reply:
Fine by me.

Reply:
Yep. It is working for me now. 
![image](https://user-images.githubusercontent.com/1278189/105211963-25387b00-5b4d-11eb-835b-b3d66b9e6d85.png)